ldaptor/delta.py

```diff
@@ -138,7 +138,7 @@ def asLDIF(self):
 
     def asLDAP(self):
         return pureldap.LDAPModifyRequest(
-            object=str(self.dn),
+            object=self.dn,
```

Comment:
this looks like a change in the API.... it I am misreading this from the diff.

in previous version it was taking bytes and not it takes a DN object?

Reply:
I could use ``self.dn.toWire()`` in this case (and other similar cases) but ``LDAPModifyRequest`` converts ``object`` to its bytes representation when its ``toWire`` method is called (or converted to string when ``__str__`` method was used). So we can pass ``self.dn`` as it is :)